v3/lints/cabf_smime_br/lint_strict_aia_contains_internal_names.go

```diff
@@ -34,39 +35,40 @@ values for each of the following types:
 id-ad-ocsp        specifies the URI of the Issuing CA's OCSP responder.
 id-ad-caIssuers   specifies the URI of the Issuing CA's Certificate.
 
-For Strict and Multipurpose: When provided, every accessMethod SHALL have the URI scheme HTTP. Other schemes SHALL NOT be present.
 *************************************************************************/
 
 func init() {
 	lint.RegisterCertificateLint(&lint.CertificateLint{
 		LintMetadata: lint.LintMetadata{
-			Name:          "w_smime_strict_aia_contains_internal_names",
-			Description:   "SMIME Strict certificates authorityInformationAccess When provided, every accessMethod SHALL have the URI scheme HTTP. Other schemes SHALL NOT be present.",
+			Name:          "w_smime_aia_contains_internal_names",
```

Comment:
The filename of this lint and test should be renamed to match this updated lint name.

Reply:
renamed